apps/web/components/v2/eventtype/EventLimitsTab.tsx

```diff
@@ -114,13 +114,99 @@ export const EventLimitsTab = (props: Pick<EventTypeSetupInfered, "eventType">)
         </div>
       </div>
       <div className="flex flex-col space-y-4 pt-4 lg:flex-row lg:space-y-0 lg:space-x-4">
-        <div className="w-full">
-          <MinutesField
-            required
-            label={t("minimum_booking_notice")}
-            type="number"
-            placeholder="120"
-            {...formMethods.register("minimumBookingNotice", { valueAsNumber: true })}
+        <div className="flex w-full items-end">
+          <Controller
+            name="minimumBookingNotice"
+            control={formMethods.control}
+            render={() => {
+              const durationArray = [eventType.minimumBookingNoticeType];
+              const minBookingValue = formMethods.watch("minimumBookingNotice");
+              let newMinBookingValue: number;
+              const convertToNewDurationType = function (
```

Comment:
Could you define this function definition outside of this render method, or maybe even move it to `packages/lib`?  There's no reason for this function to be redefined on every render, plus we keep the renders a bit clearer. 

And you will also notice that you use the `eventType.minimumBookingNoticeType` inside your function, instead of expecting it as an argument. Using these parameters outside of your function can have very unpredictable side effects, so I will really advice against doing that, and use them as an argument instead.